src/controllers/dlgprefcontroller.cpp

```diff
@@ -191,6 +192,12 @@ DlgPrefController::DlgPrefController(
         m_ui.labelHidUsagePageValue->setVisible(true);
         m_ui.labelHidUsage->setVisible(true);
         m_ui.labelHidUsageValue->setVisible(true);
+
+        // Create HID report tabs
+        m_hidReportTabsManager =
+                std::make_unique<ControllerHidReportTabsManager>(
+                        m_ui.controllerTabs, hidController);
+        m_hidReportTabsManager->createReportTypeTabs();
```

Comment:
Shall we get this tab only displayed if Mixxx is ran with `--developer`?

Reply:
This feature ist similar to the MIDI mapping tabs, which are always visible as well. If we use a switch, it should be --controller-debug not --developers , as this feature targets mapping developers.